src/mixins/AreaMixin.js

```diff
@@ -40,7 +43,7 @@ export const AreaMixinBase = types
     },
 
     get texting() {
-      return self.results.find(r => r.type === 'textarea' && r.hasValue);
+      return isAlive(self) && self.results.find(r => r.type === 'textarea' && r.hasValue);
```

Comment:
Semantically it should be `undefined`, yeah. But here I didn't care about returned value 'cause `isAlive` is here just to prevent errors on the stage of destroying storage nodes. I mean, we event will not see the result of rendering in that case.

Reply:
But logically `hasValue` also returns `boolean`... And logically on the stage of destroying `false` is also OK 'cause there is still no texting.